frontend/src/components/alerts/AlertDisplay.vue

```diff
@@ -58,7 +58,8 @@ export default {
 </script>
 
 <style scoped>
-#alert-contaier {
+#alert-container {
   z-index: 10;
+  font-size: 1.5em;
```

Comment:
This is certainly a correct solution. Maybe I can interest you in a less explicit approach?

Instead of directly defining the CSS, we could add [`fs-4`](https://getbootstrap.com/docs/5.3/utilities/text/#font-size) to the classes of `alert-container`. Would you like to give it a try?

Reply:
Ah yes thanks, i was looking for something like that, so that is then using bootstrap if i'm correct ?

Reply:
Yes, bootstrap5 with the SUSE theme: https://github.com/crazyscientist/suse-bootstrap5-theme